server.js

```diff
@@ -1,10 +1,54 @@
 const express = require("express");
 const serveStatic = require("serve-static");
+const multer = require("multer");
+const crypto = require("crypto");
+const AWS = require("aws-sdk");
+const cors = require("cors");
 
 module.exports = function() {
   const port = process.env.C20_PORT ? Number(process.env.C20_PORT) : 8080;
+  const bucketName = process.env.C20_S3_BUCKET || "reclaimers-data-test";
+  const s3 = new AWS.S3();
+
+  const saveSurvey = function(surveyData, cb) {
+    const s3Params = {Bucket: bucketName, Key: "survey-results.json"};
+    s3.getObject(s3Params, (err, data) => {
+      if (err && err.statusCode != 404) {
+        return cb(err);
+      }
+      const results = err ? {} : JSON.parse(data.Body.toString());
+      results[surveyData.ipHash] = surveyData;
+      const putParams = {
+        ...s3Params,
+        Body: JSON.stringify(results),
+        "ContentType": "application/json",
+        "ACL": "public-read"
+      };
+      s3.putObject(putParams, (err) => {
+        cb(err);
+      });
+    });
+  };
+
   const app = express();
   app.use("/", serveStatic("./dist"));
+
+  app.post("/survey/submit", cors(), multer().none(), (req, res) => {
```

Comment:
The `cors()` middleware is used because I don't really care what origin makes a POST to the API server (e.g. I want to hit it from localhost) and the impact of any abuse is low. The `multer()` middleware just parses the multipart form submissions into a nice JS object for us. The `none()` means it won't accept file uploads.